### ISSUE TITLE
[BUG] brut.androlib.err.CantFindFrameworkResException: Can't find framework resources for package of id: 125

### ISSUE THREAD

**Issue by aureq** [iBotPeaches/Apktool#2699]
### Information
1. **Apktool Version (`apktool -version`)** - 2.6.0
2. **Operating System (Mac, Linux, Windows)** - Linux, Debian 11
3. **APK From? (Playstore, ROM, Other)** - Playstore
4. **APK Link?** - [au.gov.vic.service.digitalwallet.citizen](https://play.google.com/store/apps/details?id=au.gov.vic.service.digitalwallet.citizen) 1.15.1-2808

### Stacktrace/Logcat

```
Nov 23, 2021 9:10:59 AM brut.androlib.res.decoder.AXmlResourceParser getAttributeValue
WARNING: Could not decode attr value, using undecoded value instead: ns=android, name=resource, value=0x7d040000
brut.androlib.err.CantFindFrameworkResException: Can't find framework resources for package of id: 125
	at brut.androlib.res.AndrolibResources.getFrameworkApk(AndrolibResources.java:817)
	at brut.androlib.res.AndrolibResources.loadFrameworkPkg(AndrolibResources.java:108)
	at brut.androlib.res.data.ResTable.getPackage(ResTable.java:81)
	at brut.androlib.res.data.ResTable.getResSpec(ResTable.java:64)
	at brut.androlib.res.data.ResTable.getResSpec(ResTable.java:60)
	at brut.androlib.res.data.value.ResReferenceValue.getReferent(ResReferenceValue.java:59)
	at brut.androlib.res.data.value.ResReferenceValue.encodeAsResXml(ResReferenceValue.java:45)
	at brut.androlib.res.data.value.ResScalarValue.encodeAsResXmlAttr(ResScalarValue.java:44)
	at brut.androlib.res.decoder.ResAttrDecoder.decode(ResAttrDecoder.java:44)
	at brut.androlib.res.decoder.AXmlResourceParser.getAttributeValue(AXmlResourceParser.java:384)
	at brut.androlib.res.decoder.AndroidManifestResourceParser.getAttributeValue(AndroidManifestResourceParser.java:39)
	at org.xmlpull.v1.wrapper.classic.XmlPullParserDelegate.getAttributeValue(XmlPullParserDelegate.java:69)
	at org.xmlpull.v1.wrapper.classic.StaticXmlSerializerWrapper.writeStartTag(StaticXmlSerializerWrapper.java:267)
	at org.xmlpull.v1.wrapper.classic.StaticXmlSerializerWrapper.event(StaticXmlSerializerWrapper.java:211)
	at brut.androlib.res.decoder.XmlPullStreamDecoder$1.event(XmlPullStreamDecoder.java:80)
	at brut.androlib.res.decoder.XmlPullStreamDecoder.decode(XmlPullStreamDecoder.java:139)
	at brut.androlib.res.decoder.XmlPullStreamDecoder.decodeManifest(XmlPullStreamDecoder.java:151)
	at brut.androlib.res.decoder.ResFileDecoder.decodeManifest(ResFileDecoder.java:159)
	at brut.androlib.res.AndrolibResources.decodeManifestWithResources(AndrolibResources.java:193)
	at brut.androlib.Androlib.decodeManifestWithResources(Androlib.java:140)
	at brut.androlib.ApkDecoder.decode(ApkDecoder.java:109)
	at brut.apktool.Main.cmdDecode(Main.java:175)
	at brut.apktool.Main.main(Main.java:78)
```

### Steps to Reproduce
1. `apktool -v decode --match-original --output "unpacked/universal/" "input.apk"`

### APK
https://home.menfin.net/downloads/apk.zip (I'll try to attach it to this issue but GH seems to be having troubles at the moment.)

### Notes
* Running `apktool d` doesn't show the error, *however* some resources are missing for the extraction/decode just like with the command mentioned above.
* I could be wrong, but `res/xml/splits0.xml` may be the file/resource causing the exception above.
* During build, line 134 in `AndroidManifest.xml` has to be commented (`<meta-data android:name="com.android.vending.splits" android:resource="@2097414144"/>`) which may be related to the resource mentioned just above.
* Diffing the `res/` directory between a simple `unzip` and `apktool`, a lot of resources are missing (others have different folder names though). So I guess `apktool` crashes and stops unpacking the resources
* I did build the app again, but it is significantly lower in size (15MB vs 41MB) and the app crashes.

Happy to provide additional information and/or debugs or happy to test any fix.

Thank you.

**Comment by iBotPeaches** [comment]
Thanks for a detailed bug report. I'm a bit behind on tickets, but this one in my inbox is very detailed and I don't often get the template followed with all details.

**Comment by aureq** [comment]
Thank you @iBotPeaches 

I think it's only normal to follow your template given you already give so much to the community with your tools.

If you need anything from me, let me know.

**Comment by iBotPeaches** [comment]
So this is a semi-known issue that we have no true solution for at this time. This application is packing many other packages inside.

```
➜  2699 aapt d resources Service\ Victoria_1.15.1-2808-citizen_apkcombo.com.apk | grep package
Package Group 0 id=0x7d packageCount=1 name=au.gov.vic.service.digitalwallet.citizen.check_in
Package Group 1 id=0x7e packageCount=1 name=au.gov.vic.service.digitalwallet.citizen.app
Package Group 2 id=0x7f packageCount=1 name=au.gov.vic.service.digitalwallet.citizen
```
Apktool for since ever has assumed the private package (0x7f) was internal and all other packages (0x01, etc) are frameworks elsewhere. This is changing with split applications and bundles, etc.

I began work on this in https://github.com/iBotPeaches/Apktool/pull/2517, which was to fix #2514, but ran into many issues with it. I have this application and research now, so going to mark as duplicate as #2514.

**Comment by aureq** [comment]
Sure thing @iBotPeaches and thank you for taking the time to research and reply to this issue. 👍
